flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
I can't test the other archs.

Reply:
1. it's python
2. nobody will if it is not built.

Reply:
Fair enough, removed the flathub.json.